### PR TITLE
docs: correct compile and model typo, etc

### PR DESCRIPTION
**Pull request by taorye**
Co-authored-by: Dayoung Lee <dayoung.lee@samsung.com>
ONE-vscode-DCO-1.0-Signed-off-by: taorye <taorye@outlook.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
@taorye , thank you for your contribution.
Please fill in some commit message that explains this change and with https://github.com/Samsung/ONE-vscode/wiki/ONE-vscode-Developer%27s-Certificate-of-Origin

**Comment by jyoungyun** [comment]
@taorye You can add the `one-vscode` as reviewers. :)

File: README.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ Through these activities, ONE-vscode can provide a differentiation that cannot b
 ## Quick Links
 
 - [Tutorial](./docs/Tutorial.md)
-  - The tutorial contains how to install ONE-vscode extension, how to installa toolchain, how to copmile a nodel, etc.
+  - The tutorial contains how to install ONE-vscode extension, how to installa toolchain, how to copmile a model, etc.
```
Comment (dayo09) [inline README.md:20]:
```suggestion
  - The tutorial contains how to install the ONE-vscode extension, how to install a toolchain, how to compile a model, etc.
```

**Comment by dayo09** [comment]
Thanks for the commit!

BTW, could you change the commit message with the format of commit title and body?

Example from https://github.com/Samsung/ONE-vscode/pull/1499/commits
```
[CircleGraph] Use prettier ignore view-sidebar.js   <---- Commit title

This will let prettier ignore previous code of view-sidebar.js without formatting
for better readability.  <----- Commit body

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com> <--- Your sign-off (you did well!)
```

I know it's not written in [the ONE DCO page](https://github.com/Samsung/ONE-vscode/wiki/ONE-vscode-Developer%27s-Certificate-of-Origin) but it's a convention that we are using.

I may suggest,

```
Fix typo in README.md

This commit corrects compile and model typo, etc.

Co-authored-by: Dayoung Lee [dayoung.lee@samsung.com](mailto:dayoung.lee@samsung.com)
ONE-vscode-DCO-1.0-Signed-off-by: taorye [taorye@outlook.com](mailto:taorye@outlook.com)
```

**Comment by taorye** [comment]
@dayo09 Thanks for your mention, I have changed them as your advice.

It's so sudden that I have just found this PR is still waiting for me :)

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]

**Pull request merged**